Sources/RxMoya/RxMoyaProvider.swift

```diff
@@ -69,3 +74,10 @@ public extension RxMoyaProvider {
         }
     }
 }
+
+public extension MoyaProvider {
+    
+    public var rx: RxMoyaProvider<Target> {
```

Comment:
I am not quite sure what this does / how that would work? Would this drop `RxMoyaProvider` completely, add the `.rx` var and make only `request()` be enabled for the returned value of `.rx`?